### PR TITLE
[14.0][MIG] mgmtsystem nonconformity product

### PR DESCRIPTION
**Pull request by mymage**
Migration to 14.0

### CODE REVIEW

**Comment by mymage** [comment]
@marcelofrare @primes2h pleare review

File: mgmtsystem_nonconformity_product/models/mgmtsystem_nonconformity_product.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Copyright 2019 Marcelo Frare (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+# Copyright 2019 Stefano Consolaro (Ass. PNLUG - Gruppo Odoo <http://odoo.pnlug.it>)
+
+from odoo import fields, models
+
+
+class MgmtsystemMgmProduct(models.Model):
```
Comment (max3903) [inline mgmtsystem_nonconformity_product/models/mgmtsystem_nonconformity_product.py]:
```suggestion
class MgmtsystemNonconformity(models.Model):
```

Comment (max3903) [reply mgmtsystem_nonconformity_product/models/mgmtsystem_nonconformity_product.py]: @mymage Please rename this file to `mgmtsystem_nonconformity.py` to match the model being inherited.

**Review by marcelofrare — APPROVED** [review]
runboat is ok

**Review by primes2h — CHANGES_REQUESTED** [review]
Please fixup administrative commits with the previous commit for reducing noise. 
See https://github.com/OCA/maintainer-tools/wiki/Merge-commits-in-pull-requests#mergesquash-the-commits-generated-by-bots-or-weblate

**Comment by mymage** [comment]
@max3903 @primes2h done

**Comment by primes2h** [comment]
> @max3903 @primes2h done

Not enough, you have to fixup these too.

immagine

**Review by max3903 — APPROVED** [review]

**Review by primes2h — APPROVED** [review]
LGTM, thanks!

**Review by etobella — APPROVED** [review]
/ocabot merge nobump

**Pull request merged**